registry/tests/mail_test.py

```diff
@@ -47,3 +49,10 @@ def testLinkWorksCorrectly(self, send_email):
         body = call['body']
         assert html.find(expected_test_link) != -1
         assert body.find(expected_test_link) != -1
+
+        expected_test_link = 'http://localhost:3000/test_user/pkg/comments'
```

Comment:
we should check for the presence of `commenting_user` as well

Reply:
done